lighthouse-core/audits/manifest-background-color.js

```diff
@@ -39,19 +41,24 @@ class ManifestBackgroundColor extends Audit {
   static hasBackgroundColorValue(manifest) {
     return manifest !== undefined &&
       manifest.background_color !== undefined &&
-      manifest.background_color.value !== undefined;
+      manifest.background_color.value;
```

Comment:
for what it's worth, `manifest.background_color !== undefined` isn't necessary here